apps/editing-toolkit/editing-toolkit-plugin/error-reporting/index.js

```diff
@@ -18,6 +19,12 @@ function activateSentry() {
 		release: 'wpcom-test-01',
 	} );
 
+	// Capture exceptions from Gutenberg React Error Boundaries
+	addAction( 'editor.ErrorBoundary.errorLogged', 'etk/error-reporting', ( error ) => {
```

Comment:
Will this work on Atomic if gutenberg is disabled? (I assume it would, as long as `addAction` doesn't crash everything if the action doesn't exist.)

Reply:
Atomic doesn't activate this module yet, but if it did, I don't see why it would crash -- if AT didn't have Gutenberg (and if the block editor still doesn't have the call to `doAction`) then the callback will just be registered here but never triggered.

Reply:
Was that what you had in mind?

Reply:
I think so, yeah. I was wondering what `addAction` does if the given "action type" doesn't exist

Reply:
Since it's not loaded (I keep forgetting that!) I agree we don't need to worry about it

Reply:
> I think so, yeah. I was wondering what addAction does if the given "action type" doesn't exist

I don't think it implements such a check -- though I agree it'd be nice to double check, though! I'll check this out in the following days.